components/places/src/api/matcher.rs

```diff
@@ -35,12 +35,7 @@ pub fn search_frecent(conn: &PlacesDb, params: SearchParams) -> Result<Vec<Searc
         &[
             // Try to match on the origin, or the full URL.
             &OriginOrUrl::new(&params.search_string, conn),
-            // After the first result, try the queries for adaptive matches and
-            // suggestions for bookmarked URLs.
-            &Adaptive::new(&params.search_string, conn),
-            &Suggestions::new(&params.search_string, conn),
-            // If we don't have enough results, query adaptive matches and
-            // suggestions again, matching anywhere instead of on boundaries.
+            // query adaptive matches and suggestions again, matching anywhere.
```

Comment:
Comment nit: IIUC, we don't query these "again" anymore; we just use `Anywhere`, and don't try matching on boundaries at all.